### PR TITLE
Simple changes to the top-level requirements

### PR DESCRIPTION
**Pull request by AdamuKaapan**
- Added keeping track of position at all times to manual mode as well
 - Added monitoring for Mode-Switch messages to manual mode as well

This commit is to make the top-level requirements a bit more complete,
adding things that I think were missed in my opinion. The position
tracking I added because it'll make transitioning back into autonomous
easier (which the ability to do is in the requirements already). The
mode-switch monitoring in manual is mainly just to make it consistent
between autonomous and manual mode.

(this commit is also just to give me an excuse to test using this style of Git workflow)

### CODE REVIEW

**Review by RyanBerge — DISMISSED** [review]

File: Top-Level Requirements.md [hunk 1]
```diff
@@ -25,14 +25,16 @@ Top-Level System Requirements
   * Upon receiving a “Mode-Switch” command, switch to Manual Mode
 * While in Manual Mode, the Robot **shall**:
   * Monitor network traffic for controller information
+  * At all times, keep track of its position
+  * Monitor network traffic for a “Mode-Switch” command
```
Comment (Jebbs) [inline Top-Level Requirements.md]: Since this is listed in both, why not put it in the "While in either mode" section?

Comment (collinsc) [reply Top-Level Requirements.md]: I agree with jeremy about moving the wording of keep track of position, but I would phrase it as maintain localization, that is the language we have used for the rest of requirements.

I think keeping the language as "Monitor network traffic for controller information" is more prudent. What if we wanted to transmit more kinds of commands than "Mode Switch"? I am not saying get rid of it. I am saying let's keep the more general wording as top level, and make several specific sub requirements.

**Review by collinsc — CHANGES_REQUESTED** [review]
Clean up the wording and placement a little. See comments.

File: Top-Level Requirements.md (hunk 1 above)
Comment (AdamuKaapan) [reply Top-Level Requirements.md]: I agree with all 3 points of those, lemme change that

Comment (AdamuKaapan) [reply Top-Level Requirements.md]: I actually didn't even see the either mode part before, so that's useful.

**Review by Jebbs — APPROVED** [review]

**Review by RyanBerge — APPROVED** [review]

**Pull request merged**